### PR TITLE
docs: add first week & forest survival

### PR DESCRIPTION
**Pull request by RoyalFox2140**
#### Summary

SUMMARY: [Infrastructure] "Add two documents to the doc site"

#### Purpose of change

I'd like to add new documents to the doc site, @scarf005 checking in this test to make sure this is the correct file location

#### Describe the solution

Adds the two files.

#### Describe alternatives you've considered

Letting the arcane secrets of Bright Nights remain forbidden to new practitioners.

#### Testing

If this finds a way to break the releases I'd be impressed on a cosmic level

#### Additional context

N/A

### CODE REVIEW

**Review by scarf005 — COMMENTED** [review]
For survival guides, `doc/src/content/docs/en/game` could be a good choice, as this path is intended for gameplay related pages (lore, guide, mod links).

**Comment by scarf005** [comment]
I'll also keep it as draft until the docs are complete.

**Comment by RoyalFox2140** [comment]
> For survival guides, `doc/src/content/docs/en/game` could be a good choice, as this path is intended for gameplay related pages (lore, guide, mod links).

I see this now and that's my bad since I was going off the video guide. I'm going to move them and keep working on it. I wanted to make sure I at least got the format and location correct before I lost any work

**Comment by scarf005** [comment]
yeah, the format's good.

File: doc/src/content/docs/en/game/first_week.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+**In order to maximize your ability to learn and chances of survival, Evac shelter is recommended and expected**
+
+While daunting, the game is a lot easier than some might think. You can do a great deal to prepare yourself for combat, but remember the zombies always get tougher thanks to evolution. It only gets more dangerous.
+
+# First steps
+
+* Makeshift crowbar
+* Cudgel (or other weapons)
+* Clothes
```
Comment (scarf005) [inline doc/src/content/docs/en/game/first_week.md:13]:
```suggestion
- Makeshift crowbar
- Cudgel (or other weapons)
- Clothes
```

could we use `-` over `*` or `+` in favor of consistency? (applies to other unordered lists)

File: doc/src/content/docs/en/game/first_week.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: first week
+---
+
+**In order to maximize your ability to learn and chances of survival, Evac shelter is recommended and expected**
+
+While daunting, the game is a lot easier than some might think. You can do a great deal to prepare yourself for combat, but remember the zombies always get tougher thanks to evolution. It only gets more dangerous.
+
+# First steps
```
Comment (scarf005) [inline doc/src/content/docs/en/game/first_week.md:9]:
```suggestion
## First steps
```

due to limitation in starlight docs, heading level 1 (x1 `#`) will not be shown in table of contents.

| `#` headings | `##` headings |
|--------|--------|
| image | image |

it's a solid choice to only use heading h2(`##`) and h3(`###`), as using h4(`####`) might mean the article is getting more complex. 

it's also a good idea to split the article into multiple smaller ones in such cases.

File: doc/src/content/docs/en/game/first_week.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: first week
+---
+
+**In order to maximize your ability to learn and chances of survival, Evac shelter is recommended and expected**
```
Comment (scarf005) [inline doc/src/content/docs/en/game/first_week.md:5]: image

```suggestion
:::tip

In order to maximize your ability to learn and chances of survival, Evac shelter is recommended and
expected.

:::
```

you can use [aside](https://starlight.astro.build/guides/authoring-content/#asides) to make them look pretty.

File: doc/src/content/docs/en/game/first_week.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+If you've started in the evac shelter, grab a rock outside and smash a locker and a bench. A makeshift crowbar will help you open doors quietly and just takes a pipe and a tool with hammering.
+
+You need a good weapon, this is usually the cudgel in the early game. If you didn't start with a knife, you can always make a stone knife with no skills. the makeshift crowbar lacks the hit bonus the cudgel has, which is why I recommend it. Optionally you can get a wooden spear but it takes some character skills and a fire. A self bow and crude arrows is weak but is not a bad opener to combat, any damage helps.
+
+You likely won't start with the perfect insulation from the weather, so you need to manage your temperature as slowdown penalties from the heat or cold will kill you quickly via slowing you down in combat. If its too hot this is easy you just take off layers. If it's too cold you'll need to craft what clothes you can or loot what you can but don't waste too much time on this. You'll likely find the clothes you need.
+
+**You want to minimize how many enemies you take on at once until you learn the game and the limits of your character, usually 1 enemy at a time is ideal**
```
Comment (scarf005) [inline doc/src/content/docs/en/game/first_week.md:21]:
```suggestion
:::note

You want to minimize how many enemies you take on at once until you learn the game and the limits of your character, usually 1 enemy at a time is ideal

:::
```

the caveat of using aside syntax is that
1. this isn't made markdown standard yet
2. formatter will understand it as plaintext and  asides without newlines between `:::` gets formatted weirdly
3. therefore, newlines are needed between `:::`

File: doc/src/content/docs/en/game/first_week.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+* Makeshift crowbar
+* Cudgel (or other weapons)
+* Clothes
+
+If you've started in the evac shelter, grab a rock outside and smash a locker and a bench. A makeshift crowbar will help you open doors quietly and just takes a pipe and a tool with hammering.
+
+You need a good weapon, this is usually the cudgel in the early game. If you didn't start with a knife, you can always make a stone knife with no skills. the makeshift crowbar lacks the hit bonus the cudgel has, which is why I recommend it. Optionally you can get a wooden spear but it takes some character skills and a fire. A self bow and crude arrows is weak but is not a bad opener to combat, any damage helps.
+
+You likely won't start with the perfect insulation from the weather, so you need to manage your temperature as slowdown penalties from the heat or cold will kill you quickly via slowing you down in combat. If its too hot this is easy you just take off layers. If it's too cold you'll need to craft what clothes you can or loot what you can but don't waste too much time on this. You'll likely find the clothes you need.
```
Comment (scarf005) [inline doc/src/content/docs/en/game/first_week.md:19]:
```suggestion
### Makeshift crowbar

If you've started in the evac shelter, grab a rock outside and smash a locker and a bench. A makeshift crowbar will help you open doors quietly and just takes a pipe and a tool with hammering.

### Cudgel (or other weapons)

You need a good weapon, this is usually the cudgel in the early game. If you didn't start with a knife, you can always make a stone knife with no skills. the makeshift crowbar lacks the hit bonus the cudgel has, which is why I recommend it. Optionally you can get a wooden spear but it takes some character skills and a fire. A self bow and crude arrows is weak but is not a bad opener to combat, any damage helps.

### Clothes

You likely won't start with the perfect insulation from the weather, so you need to manage your temperature as slowdown penalties from the heat or cold will kill you quickly via slowing you down in combat. If its too hot this is easy you just take off layers. If it's too cold you'll need to craft what clothes you can or loot what you can but don't waste too much time on this. You'll likely find the clothes you need.
```

using headings have advangtages of generating table of contents automatically.

## Before

image

## After

image

**Review by scarf005 — COMMENTED** [review]
Congrats for joining the first contributor for the revamped docs. i've made change requests regarding formatting (that i missed writing on docs.md). if you're okay with it, i'll format, re-organize and approve them tomorrow.

**Comment by RoyalFox2140** [comment]
> Congrats for joining the first contributor for the revamped docs. i've made change requests regarding formatting (that i missed writing on docs.md). if you're okay with it, i'll format, re-organize and approve them tomorrow.

Please add all formatting things (heading 1's can't be used, we're going with - instead of * or +) in the guide for contributing docs.

File: doc/src/content/docs/en/game/first_week.md (hunk 5 above)
Comment (RoyalFox2140) [reply doc/src/content/docs/en/game/first_week.md]: I guess it is more readable this way, I was showing them as bullet points to keep it as a task list for new players to follow

**Review by scarf005 — APPROVED** [review]
## Rendered pages
<details><summary>First week
</summary>

image

</details> 

<details><summary>Forest survival
</summary>

image

</details> 

I made some edits to your guides, is this OK with you?
Should i rollback checklist?

**Comment by RoyalFox2140** [comment]
> ## Rendered pages
> First week
> Forest survival
> I made some edits to your guides, is this OK with you? Should i rollback checklist?

That may be going with too many sections but if its more readable then it looks good to me

**Comment by RoyalFox2140** [comment]
I was concerned with adding too many headings and sections due to making the pages extremely long and hard to read through, but I understand making large paragraphs is not very readable either. If we go with many sections and headings that's ok with me, but we will want to add this officially to the style guide for docs

**Pull request merged**